src/Screens/Main/BuddyList/BannedList.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import RN from 'react-native';
+import { useSelector } from 'react-redux';
+
+import * as navigationProps from '../../../lib/navigation-props';
+
+import * as buddyState from '../../../state/reducers/buddies';
+
+import colors from '../../components/colors';
+import fonts from '../../components/fonts';
+import { textShadow } from '../../components/shadow';
+import Message from '../../components/Message';
+import RemoteData from '../../components/RemoteData';
+import TitledContainer from '../../components/TitledContainer';
+
+import Button from './Button';
+
+import { ChatRoute } from '../Chat';
+
+export type BannedListRoute = {
+  'Main/BuddyList/BannedList': {};
+};
+
+type Props = navigationProps.NavigationProps<BannedListRoute, ChatRoute>;
+
+export default ({ navigation }: Props) => {
+  const buddies = useSelector(buddyState.getBannedBuddies);
+
+  const onPress = (buddyId: string) => {
+    navigation.navigate('Main/Chat', { buddyId });
+  };
+
+  const onPressBack = () => {
+    navigation.goBack();
+  };
+
+  return (
+    <TitledContainer
+      TitleComponent={
+        <RN.View style={styles.header}>
+          <RN.TouchableOpacity
+            style={styles.backButtonTouchable}
+            onPress={onPressBack}
+          >
+            <RN.Image
+              source={require('../../images/chevron-left.svg')}
+              style={styles.backButtonIcon}
+            />
+          </RN.TouchableOpacity>
+          <Message
+            id="main.chat.navigation.banned"
+            style={styles.screenTitleText}
+          />
+        </RN.View>
+      }
+      color={colors.blue}
+    >
+      <RemoteData data={buddies} fetchData={() => {}}>
+        {value => (
+          <RN.ScrollView
+            style={styles.scrollView}
+            contentContainerStyle={styles.scrollContent}
+            testID={'main.buddyList.view'}
+          >
+            {value.map(buddy => (
```

Comment:
value is not the best name for buddyList : ) maybe it could be buddies or something